app/app.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     compile(group = "com.google.code.gson", name = "gson", version = "2.8.1")
     fun testFx(name: String, version: String = "4.0.+") =
         create(group = "org.testfx", name = name, version = version)
+    runtime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "+", classifier = "all")
```

Comment:
http://first.wpi.edu/FRC/roborio/maven/development/edu/wpi/first/ntcore/ntcore-java/3.1.7-20170808143930-12-gccfeab5/

This would be the version we would have to put in manually now.